app/page.tsx

```diff
@@ -3,10 +3,12 @@ import SectionOne from "@/app/Components/SectionOne/SectionOne";
 import SectionTwo from "@/app/Components/SectionTwo/SectionTwo";
 import SectionThree from "@/app/Components/SectionThree/SectionThree";
 import SectionFour from "@/app/Components/SectionFour/SectionFour";
+import HeaderMenu from "@/app/Components/header/Header";
 
 export default function Home() {
     return (
         <main className={s.main}>
+            <HeaderMenu/>
             <SectionOne/>
```

Comment:
Please rename this section(Number) for more telling names. Like: ContactSection, InviteSection etc.

Reply:
done